lib/validator/validator_test.go

```diff
@@ -261,16 +261,33 @@ func init() {
 			}, false),
 
 		// (API) PoolMetadata
-		Entry("should accept IP pool with IPv4 CIDR /26", api.IPPoolMetadata{CIDR: netv4_3}, true),
-		Entry("should accept IP pool with IPv4 CIDR /10", api.IPPoolMetadata{CIDR: netv4_4}, true),
-		Entry("should accept IP pool with IPv6 CIDR /122", api.IPPoolMetadata{CIDR: netv6_3}, true),
-		Entry("should accept IP pool with IPv6 CIDR /10", api.IPPoolMetadata{CIDR: netv6_4}, true),
-		Entry("should reject IP pool with IPv4 CIDR /27", api.IPPoolMetadata{CIDR: netv4_5}, false),
-		Entry("should reject IP pool with IPv6 CIDR /128", api.IPPoolMetadata{CIDR: netv6_1}, false),
+		Entry("should accept IP pool with IPv4 CIDR /26", api.IPPool{Metadata: api.IPPoolMetadata{CIDR: netv4_3}}, true),
+		Entry("should accept IP pool with IPv4 CIDR /10", api.IPPool{Metadata: api.IPPoolMetadata{CIDR: netv4_4}}, true),
+		Entry("should accept IP pool with IPv6 CIDR /122", api.IPPool{Metadata: api.IPPoolMetadata{CIDR: netv6_3}}, true),
+		Entry("should accept IP pool with IPv6 CIDR /10", api.IPPool{Metadata: api.IPPoolMetadata{CIDR: netv6_4}}, true),
+		Entry("should accept a disabled IP pool with IPv4 CIDR /27",
```

Comment:
It's not really an invalid pool size is why :)

We only limit to a `/26` because that is the minimum block size in Calico IPAM.  If the Pool isn't being used for IPAM, then there's no reason to require a certain size.